support-frontend/assets/components/subscriptionCheckouts/thankYou/returnSection.tsx

```diff
@@ -16,11 +19,11 @@ const buttonStyles = css`
 	}
 `;
 type PropTypes = {
-	subscriptionProduct: SubscriptionProduct;
+	subscriptionProduct: SubscriptionProduct | typeof DigitalPackAddOn;
```

Comment:
Is there a reason we didn't add DigitalPackAddOn to type SubscriptionProduct (used quite widely so would have to change a lot?)